pkg/lakom/verifysignature/verifysignature_suite_test.go

```diff
@@ -5,13 +5,187 @@
 package verifysignature_test
 
 import (
+	"crypto"
```

Comment:
In this file the feedback would be the same as the `resolvetag_suite_test.go`, thus I will skip it to avoid duplication.